contracts/governance/extensions/GovernorTimelockCompound.sol

```diff
@@ -145,10 +142,12 @@ abstract contract GovernorTimelockCompound is IGovernorTimelock, Governor {
 
         uint256 eta = proposalEta(proposalId);
         if (eta > 0) {
+            // update state first
+            delete _proposalTimelocks[proposalId];
+            // do external call later
```

Comment:
Why is this?

Reply:
now that `cancel` is public, slither detects a possible "reentrancy attack" here because `_timelock.cancelTransaction` is an external call that is done before a state change.

In fact I don't think there is any risk, because `super._cancel` does the reentrency protection already.


It just to calm slither